ejercicio1vulky/src/main/java/com/mycompany/ejercicio1vuky/ModeloPersona.java

```diff
@@ -0,0 +1,48 @@
+
+package com.mycompany.ejercicio1vuky;
+
+
+public class ModeloPersona{
+    
+ private String colorP;
+ private  String nombre;
+ private int edad;
+
+    public String getColorP() {
+        return colorP;
+    }
+
+    public void setColorP(String colorP) {
```

Comment:
Same here, Update method name to reflect the same than attribute

Reply:
ready